documents/wnd/texturing.md

```diff
@@ -0,0 +1,137 @@
+```nasm
+ENABLEDDRAWDATA = IMAGE: LoadingBar_R, COLOR: 255 0 0 0, BORDERCOLOR: 255 128 128 0,
+                  IMAGE: LoadingBar_L, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: LoadingBar_C, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: NoImage, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: NoImage, COLOR: 47 55 168 255, BORDERCOLOR: 254 254 254 255,
+                  IMAGE: LoadingBar_DePowered, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: LoadingBar_Progress, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: NoImage, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0,
+                  IMAGE: NoImage, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0;
+```
+
+Here, `LoadingBar_R`, `LoadingBar_L` and other are the names of textures defined in the
+`SCSmShellUserInterface512.ini` file. The `NoImage` entries must still be included in their correct slots, even if there is no specific texture for that part of the control.
+
+**Important:** Each control has it's own way of setting up `DRAWDATA`, for example,  a `LoadingBar_Progress` must have the progress bar texture in slot 7, so if you try to use that slot for anything else, the progress bar will not be shown.
+
+It's important to use the example for each specific control and keep the textures on the exact slot, so the rendering will be correct, or you may cause the game to crash.
+## Creating and Modifying Textures
+
+You can create new textures or modify existing ones by editing the `.tga` files and the corresponding INI mapping files.
+
+### Step-by-Step Guide
+
+1. **Edit the `.tga` File:**
+    * Open the TGA file (`SCSmShellUserInterface512_001.tga`) with an image editor that supports TGA files (e.g.,
+      Photoshop, GIMP).
+    * Add or modify the textures as needed.
+    * Make sure the new textures do not overlap with existing ones.
+2. **Update the Mapping INI File:**
+    * Open the corresponding INI file (`SCSmShellUserInterface512.ini`).
+    * For each new texture, create a new `MappedImage` entry, as shown above, using unique `TextureName`, the correct
+      filename, `TextureWidth` and `TextureHeight` and the precise coordinates (`Coords`) of the new texture within the
+      edited TGA file.
+    * If you modified an existing texture, update its `Coords` values accordingly.
+
+3. **Test Your Changes:**
+    * Save both the edited TGA and INI files.
+    * Launch the game and check that the changes to the UI are applied correctly.
+
+### Important Notes
+
+* **Coordinate Precision**: Be accurate when recording the texture coordinates in the INI file.
+* **Texture Overlap**: Avoid having textures that overlap in the TGA file.
+* **Backup:** Always back up the original files before editing.
+* **TGA Format**: Save your images as a 32 bit uncompressed TGA file.
```

Comment:
In Patch development we use PSD or TIFF files. TGA saves no layers.

Also, it is possible to reference DDS DXT1, DXT5 in WND.